lang/cs/Org.Apache.REEF.Tests/Network/GroupCommunicationTests.cs

```diff
@@ -696,8 +696,7 @@ public static List<ICommunicationGroupClient> CommGroupClients(string groupName,
             IPEndPoint nameServerEndpoint, IObserver<NsMessage<GroupCommunicationMessage>> handler)
         {
             return new NetworkService<GroupCommunicationMessage>(
-                0, nameServerEndpoint.Address.ToString(), nameServerEndpoint.Port, 
-                handler, new StringIdentifierFactory(), new GroupCommunicationMessageCodec());
+                0, handler, new StringIdentifierFactory(), new GroupCommunicationMessageCodec(), new NameClient(nameServerEndpoint.Address.ToString(), nameServerEndpoint.Port));
```

Comment:
No need to change, but you *could* use tang to create the instances for tests. It is one of the major upsides of having dependency injection in the first place.